subsys/bluetooth/audio/mcc.c

```diff
@@ -159,6 +159,15 @@ int on_icon_content(struct bt_ots_client *otc_inst,
 		    uint32_t len, uint8_t *data_p, bool is_complete);
 #endif /* CONFIG_BT_MCC_OTS */
 
+static struct mcs_instance_t *lookup_inst_by_conn(struct bt_conn *conn)
```

Comment:
Does this belong in he other commit?

Reply:
No, it is also being used by this commit